### ISSUE TITLE
The automatically checked off Walkthrough steps in vscode aren't read by NVDA screen reader and hence not accessible

### ISSUE THREAD

**Issue by Achyuth-ibm-z** [microsoft/vscode#247108]
The issue is that the information is not exposed to the user to allow for screen readers to announce events that are completed conditionally in a walkthrough step. A blind user would not know that a walkthrough step has been checked off.

Attached a video of the automatically ticked off walkthrough steps on window reload

- VS Code Version: 1.99.1
- OS Version: 15.3.2

**Comment by Achyuth-ibm-z** [comment]
I have upgraded to the latest version (1.99.3) and yet this issue persists.

**Comment by Achyuth-ibm-z** [comment]
I am trying to attach a .mov extension video of the automatically ticked off walkthrough steps on window reload, but it is failing to upload

**Comment by eleanorjboyd** [comment]
tried this with the mac built-in and am not able to get any more information via the screen reader then that I can expand it (but when I do so nothing happens). Can you clarify expected behavior and a little more clarity on what I'm missing for steps
<img width="1404" height="731" alt="Image" src="https://github.com/user-attachments/assets/d09e47b9-1b82-4aa9-ae97-8864676e7940" />

**Comment by benvillalobos** [comment]
Heard "Step: Choose your theme completed" upon clicking "Browse Color Themes"

Note: Verified on windows.
